frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -141,6 +142,17 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     combinedDisk.getStorageClassName() || '',
   );
 
+  React.useEffect(() => {
+    if (!isEditing) {
+      const defaultStorageClass = getLoadedData(storageClasses, []).find(
+        (sc) => getAnnotations(sc, {})[DEFAULT_SC_ANNOTATION] === 'true',
```

Comment:
this is already used once in the codebase. Having a `isDefaultSC(storageClass)` function would be nice

Reply:
moved to a common class